suite-native/device/src/hooks/useDeviceConnect.tsx

```diff
@@ -43,6 +73,12 @@ export const useDeviceConnect = () => {
 
     const { translate } = useTranslate();
 
+    const handleDisconnect = useCallback(() => {
+        if (!currentDevice) return;
+
+        dispatch(deviceActions.forgetDevice(currentDevice));
```

Comment:
How is this working now for you? Is portfolio tracker as `selectedDevice` after this action is triggered?

Reply:
As we discussed, i am marking this TODO and creating issue to handle this in general:
 
https://github.com/trezor/trezor-suite/issues/9920